pkg/providers/launchtemplate/launchtemplate.go

```diff
@@ -147,3 +171,21 @@ func mergeTags(tags ...map[string]string) (result map[string]*string) {
 		return strings.ReplaceAll(key, "/", "_"), to.StringPtr(value)
 	})
 }
+
+// getVnetInfoLabels returns the labels for Azure CNI
```

Comment:
// getVnetInfoLabels returns VNet info labels 

whats the point of this comment other than to pass ci? "getVnetInfoLabels returns VnetInfoLabels" isn't exactly helpful, not that "the labels for Azure CNI" is either but sometimes i wonder if requiring the comment for the sake of requiring it is worth it.

Reply:
i should probably just remove the comment

Reply:
My main point was that "Azure CNI" is not quite correct (as these labels are only used for subset of configurations), but VNet info labels  is not that much better. Would prefer to actually  capture in comments what we know about those (needed for Azure CNI overlay and pod subnet), so that we don't forget to make it properly conditional